index.html

```diff
@@ -41,116 +50,153 @@ <h1 class="intro">Hey there. I'm Oyinlade <br>
-          </div>
-          <div class="btn">
-            <a href="#">See Project</a>
-          </div>
-        </div>
+  <div class="card-container">
+    <div class="card">
+      <div class="card-content">
+      <h2>Multi-Post Stories Gain+Glory</h2>
+      <div class="tags">
+        <span>Ruby on rails</span>
+        <span>Css</span>
+        <span>Javascript</span>
+        <span>Html</span>
+      </div>
+      <div class="btn">
+        <a href="#">See Project</a>
+      </div>
+    </div>
+  </div>
+  </div>
 
+  <div class="card-container">
+    <div class="card">
+      <div class="card-content">
+      <h2>Multi-Post Stories Gain+Glory</h2>
+      <div class="tags">
+        <span>Ruby on rails</span>
+        <span>Css</span>
+        <span>Javascript</span>
+        <span>Html</span>
+      </div>
+      <div class="btn">
+        <a href="#">See Project</a>
       </div>
     </div>
+  </div>
+  </div>
 
-    <div>
-      <div class="card-container-4">
-        <div class="card-4">
-          <h2>Multi-Post Stories Gain+Glory</h2>
-          <div class="tags">
-            <span>Ruby on rails</span>
-            <span>Css</span>
-            <span>Javascript</span>
-            <span>Html</span>
-          </div>
-          <div class="btn">
-            <a href="#">See Project</a>
-          </div>
-        </div>
+  <div class="card-container">
+    <div class="card">
+      <div class="card-content">
+      <h2>Multi-Post Stories Gain+Glory</h2>
+      <div class="tags">
+        <span>Ruby on rails</span>
+        <span>Css</span>
+        <span>Javascript</span>
+        <span>Html</span>
+      </div>
+      <div class="btn">
+        <a href="#">See Project</a>
+      </div>
+    </div>
+  </div>
+  </div>
 
+  <div class="card-container">
+    <div class="card">
+      <div class="card-content">
+      <h2>Multi-Post Stories Gain+Glory</h2>
+      <div class="tags">
+        <span>Ruby on rails</span>
+        <span>Css</span>
+        <span>Javascript</span>
+        <span>Html</span>
+      </div>
+      <div class="btn">
+        <a href="#">See Project</a>
       </div>
     </div>
+  </div>
+  </div>
+  </div>
   </section>
   <!-- about me -->
   <section>
+    <div class="about-me-bg">
     <div class="about-me">
       <h2>About Me</h2>
       <p> I can help you build a product, feature or website Look through some of my work and experience! If you like
         what you see and have a project you need coded,
         don't hestiate to contact me.</p>
       <div class="resume-btn"><a href="#">Get My Resume</a></div>
     </div>
+  </div>
     <!-- languages -->
+    <div class="about-me-skils">
     <div class="languages">
       <img src="./images/icon-languages.svg" alt="">
       <h1>LANGUAGES</h1>
       <div class="lang-tags">
         <span>Javascript</span>
+        <span>Css</span>
         <span>Ruby on rails</span>
         <span>Html</span>
-        <span>Css</span>
       </div>
     </div>
```

Comment:
- [ ] You don't have spacing from the top and the background image is missing. Kindly implement it and decrease spacing after `get my resume` buttons
### your design
![your](https://user-images.githubusercontent.com/81727988/171649047-30bbe8fa-1d64-4c35-99a4-1ce7970aaaf4.png)
### Figma design
![figma](https://user-images.githubusercontent.com/81727988/171649111-4f684b65-9439-4926-998d-6cdb36113209.png)

Reply:
- This issue is well resolved :heavy_check_mark: